README.md

```diff
@@ -404,6 +404,35 @@ GET /_opendistro/_knn/HYMrXXsBSamUkcAjhjeN0w/stats/circuit_breaker_triggered,gra
 }
 ```
 
+## Warmup API
+### Overview
+The HNSW graphs used to perform k-Approximate Nearest Neighbor Search are stored as `.hnsw` files with the other Lucene segment files. In order to perform search on these graphs, they need to be loaded into native memory. If the graphs have not yet been loaded into native memory, upon search, they will first be loaded and then searched. This can cause high latency during initial queries. To avoid this, users will often run random queries during a warmup period. After this warmup period, the graphs will be loaded into native memory and their production workloads can begin. This process is indirect and requires extra effort. 
```

Comment:
Should we also call out Idempotency with the ```warmup``` api? warmup api ensures only the graphs belonging to the segments that did not get loaded are loaded to memory. Existing graphs would be untouched?

Reply:
Sure, I can call this out.